### PR TITLE
chore: set max SDK version to <3.0.0 upgrade test and remove browser …

### PR DESCRIPTION
**Pull request by eredo**
I didn't update the version in pubspec.yaml yet. If somebody provides me with the proper version, I'll add this to the PR and add an entry within the changelog.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -4,11 +4,10 @@ author: Dart Team <misc@dartlang.org>
 description: A shelf handler for proxying requests to another server.
 homepage: https://github.com/dart-lang/shelf_proxy
 environment:
-  sdk: '>=1.23.0 <2.0.0'
+  sdk: '>=1.23.0 <3.0.0'
```
Comment (kevmoo) [inline pubspec.yaml]: Update the lower bound here to `>=2.0.0 <3.0.0`

**Comment by kevmoo** [comment]
@eredo – you need to update `.travis.yml` to remove `1.23.0` SDK – then we'll be good!

**Comment by eredo** [comment]
@kevmoo - sorry for the delay, time difference ;) . I removed `1.23.0`.

**Comment by eredo** [comment]
@kevmoo any schedule on when this version will be published?

**Comment by kevmoo** [comment]
@nex3 – would you help w/ this?

**Pull request merged**